data/StateMachine.j2119

```diff
@@ -41,6 +43,8 @@ A Retrier MUST have a nonempty-string-array field named "ErrorEquals".
 A Retrier MAY have an positive-integer field named "IntervalSeconds".
 A Retrier MAY have a nonnegative-integer field named "MaxAttempts" whose value MUST be less than 99999999.
 A Retrier MAY have a float field named "BackoffRate" whose value MUST be greater than or equal to 1.0.
+A Retrier MAY have an positive-integer field named "MaxDelaySeconds".
```

Comment:
small typo: a* vs. an (seems like its spread out across the file)

Reply:
Fixing.